src/legacy/ui/public/agg_types/utils.ts

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-import { parseInterval } from '../utils/parse_interval';
+import { isValidEsInterval } from '../../../core_plugins/data/common';
```

Comment:
Are you importing from `data/common` here instead of `data/public` to avoid the mocking issues that were surfacing due to all of the stuff that's imported into `data/public/index`? 

Because otherwise I'd suggest importing from `data/public` since `common` will be an implementation detail of the plugin in the new platform, and may change.

Reply:
Yes that was exactly the fix (that I didn't remember in our last meeting). So basically because of the mocking issues I import directly from `common`. I also added to the PR description that this is a tech-debt we need to remove later on, once we have a proper solution for the mocking issue with one large entry file (or then potentially another solution).

cc @skaapgif @joshdover This was actually the workaround I used for now, which I forgot about in our last meeting, how I fixed it in this PR :) So simply deep importing from not the top level entry file.